frontend/src/styles/buttons/ScrollToTopButton.module.css

```diff
@@ -0,0 +1,23 @@
+.ScrollToTopButton {
+    position: fixed;
+    display: inline-block;
+    right: 2%;
+    bottom: 20%;
+    z-index: 99;

```

Comment:
consideration for future: why 99?

Reply:
I think i set it to 99 because I thought 100 was the z-index for the navbar (it's actually the z-index for the dropdown menu that appears when hovering on "Apply" in the navbar and the z-index for the navbar only in the About Us page implementation of the navbar). We also want the arrow button to be clickable from anywhere on any page - the teamMembersDiv on the About Us page also had a z-index of 5 and the button originally had a lower z-index so it was not clickable in that section before , so i just bumped up the z-index on the arrow button to be very high so that wouldn't be an issue again). The actual value the z-index got set to can be changed though, if it makes more sense at a different value.